src/main/java/seedu/duke/company/CompanyList.java

```diff
@@ -149,7 +155,12 @@ public void markUnconfirm(int companyNum) throws InvalidIndexException, EmptyLis
             throw new InvalidIndexException();
         }
         Company company = companyList.get(companyNum);
-        company.markUnconfirmed();
-        ui.showSuccessfulConfirmedMessage();
+        if (company.isConfirmed == false){
+            ui.showExistingUnconfirmationMessage();
+        }
+        else {
+            company.markUnconfirmed();
```

Comment:
Perhaps move the else statement to be on the same line as the closing curly braces